packages/cli/e2e-test/helpers.js

```diff
@@ -117,12 +117,29 @@ async function waitForPageWithText(
     }
   }
 
+  // The page may not be fully loaded and hence we need to retry.
+  const maxAttemptsToSearchText = 3;
```

Comment:
Think these could be `loadAttempts` and `findTextAttempts` to keep it short :p

Reply:
Yep, reads better :)